src/query/codegen/Cargo.toml

```diff
@@ -20,3 +20,5 @@ common-expression = { path = "../expression" }
 
 serde = { version = "1.0.137", features = ["derive"] }
 serde_json = "1.0.81"
+ordered-float = { git = "https://github.com/andylokandy/rust-ordered-float.git", branch = "as", features = ["serde"] }
```

Comment:
Can we publish a crate or move it to `https://github.com/datafuse-extras`?

Reply:
It's just temporary for https://github.com/reem/rust-ordered-float/pull/110

Reply:
What's the convention for such situations?

Reply:
OK， we do not recommend to use a personal git repo in databend :)